### ISSUE TITLE
question

### ISSUE THREAD

**Issue by JNU-luyi** [bio-ontology-research-group/deepgoplus#59]
![image](https://github.com/bio-ontology-research-group/deepgoplus/assets/100257990/af4bfb06-3c79-49ae-91e4-a4901cd2a160)
Based on your description, the data collected before January 2016 serves as the training set, and the experimental annotations collected between January and October are used as the test set. I have already downloaded the UniProt Swiss-Prot data files for January 2016 and October 2016 from the FTP. Is there any code available for extracting the test set from this data? I am eager to learn how this part is done. Thank you!
